feat: [#176434157] Remove payment CTA and expired badge

Undermaken opened the pull request:
## Short description
This PR removes these elements from the message list item

- payment CTA
- expired (expired & not paid payment advice) badge

### bug fix
- due to a bad logic, a spacer was rendered despite the content is empty

| before  | now |
| ------------- | ------------- |
![before](https://user-images.githubusercontent.com/822471/105329957-07612980-5bd2-11eb-8cb3-14166a1eed55.png) | ![now](https://user-images.githubusercontent.com/822471/105329978-0f20ce00-5bd2-11eb-9c33-d2444772ec56.png)

thisisjp reviewed (changes requested):
@Undermaken could we have the same height logic whether the list item has a CTA or not? I see some inconsistencies when the CTA is missing: the space between title and the bottom grey line is not needed

Undermaken commented:
> @Undermaken could we have the same height logic whether the list item has a CTA or not? I see some inconsistencies when the CTA is missing: the space between title and the bottom grey line is not needed

I fixed some UI logics where the app renders a spacer also if the content is empty. I updated the PR description with the last new screens
